config/args.js

```diff
@@ -1,7 +1,11 @@
 const commandLineArgs = require('command-line-args');
 
 const scriptNameRegex = /scripts[\/\\]([\w-]*)\.js$/i;
-const scriptName = process.argv[1].match(scriptNameRegex)[1];
+const scriptNameByRegex = process.argv[1].match(scriptNameRegex);
+
+const scriptName = scriptNameByRegex
```

Comment:
command-line-commands looks good, but I don't think it fixes the problem here where the argument is actually a path :<